### PR TITLE
Make profile ids configurable

### PR DESCRIPTION
**Pull request by monich**
This makes a number of things configurable.

### CODE REVIEW

**Review by mkosola — APPROVED** [review]

File: src/binder_data.c [hunk 1]
```diff
@@ -1272,23 +1272,27 @@ binder_data_call_setup_new(
...
-            setup->profile_id = data->mms_data_profile_id;
+            setup->profile_id = dpc->mms_profile_id;
             break;
         case OFONO_GPRS_CONTEXT_TYPE_IMS:
             setup->profile_id = RADIO_DATA_PROFILE_IMS;
             break;
         case OFONO_GPRS_CONTEXT_TYPE_ANY:
         case OFONO_GPRS_CONTEXT_TYPE_INTERNET:
         case OFONO_GPRS_CONTEXT_TYPE_WAP:
+            /* Leave the default valure untouched */
```
Comment (sledges) [inline src/binder_data.c]:
```suggestion
            /* Leave the default value untouched */
```

File: src/binder_network.c [hunk 2]
```diff
@@ -1107,7 +1113,7 @@ binder_network_set_data_profiles(
         dp = gbinder_writer_malloc0(&writer, elem * n);
         for (l = self->data_profiles, i = 0; i < n; l = l->next, i++) {
             binder_network_fill_radio_data_profile_1_4(&writer, dp + i,
-                (BinderNetworkDataProfile*) l->data);
+                 (BinderNetworkDataProfile*) l->data, dpc);
```
Comment (sledges) [inline src/binder_network.c]:
```suggestion
                (BinderNetworkDataProfile*) l->data, dpc);
```

**Review by sledges — CHANGES_REQUESTED** [review]
Only cosmetic suggestions. I would also friendly encourage to use [present tense](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) in commit messages (Made -> Make; Git itself uses present tense in its commits like Merge, Revert etc. and it's nicer when cherry-picking)

**Comment by Thaodan** [comment]
> Only cosmetic suggestions. I would also friendly encourage to use [present tense](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) in commit messages (Made -> Make; Git itself uses present tense in its commits like Merge, Revert etc. and it's nicer when cherry-picking)

Same for the PR title?
Should LTE written in caps? It is usually written that way.

**Comment by monich** [comment]
> Only cosmetic suggestions. I would also friendly encourage to use [present tense](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) in commit messages (Made -> Make; Git itself uses present tense in its commits like Merge, Revert etc. and it's nicer when cherry-picking)

I tend to use imperative tense for PR titles (to indicate an intention) and past tense for commit messages (to indicate the completion of an action once the PR gets merged). But I have to agree that this one isn't particularly consistent))

**Comment by monich** [comment]
> > Only cosmetic suggestions. I would also friendly encourage to use [present tense](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) in commit messages (Made -> Make; Git itself uses present tense in its commits like Merge, Revert etc. and it's nicer when cherry-picking)
> 
> Same for the PR title? Should LTE written in caps? It is usually written that way.

The `lte` config value is defined in lower case, the commit message refers to the config value, rather than LTE as a technology.

**Review by sledges — APPROVED** [review]

**Comment by Thaodan** [comment]
> > > Only cosmetic suggestions. I would also friendly encourage to use [present tense](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) in commit messages (Made -> Make; Git itself uses present tense in its commits like Merge, Revert etc. and it's nicer when cherry-picking)
> > 
> > 
> > Same for the PR title? Should LTE written in caps? It is usually written that way.
> 
> The `lte` config value is defined in lower case, the commit message refers to the config value, rather than LTE as a technology.

Ok makes sense then.

**Review by Thaodan — APPROVED** [review]

**Pull request merged**